frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/Loading/Loading.tsx

```diff
@@ -0,0 +1,14 @@
+import { useRive } from '@rive-app/react-canvas'
+import type { ComponentProps, FC } from 'react'
+
+type Props = ComponentProps<'canvas'>
+
+export const Loading: FC<Props> = (props) => {
+  const { RiveComponent } = useRive({
+    src: '/loading.riv',
```

Comment:
Currently it works because it is bundled with @liam-hq/cli vite, but if you want to call @liam-hq/erd-core from another part, it will stop working.
Essentially, it should be better to load via a CDN.